tox.ini

```diff
@@ -14,8 +15,10 @@ deps =
     isort
     pytest
     pytest-cov
+    validate-pyproject
 commands =
     black --check --diff .
     # Isort needs to be fixed (skip is not recognized anymore)
     # isort --check-only -rc fake_useragent --diff
     pytest tests
+    validate-pyproject pyproject.toml
```

Comment:
Put the validation above the pytest (also don't forgot the update from upstream), since we have file overlap.